scripts/us_epa/air_emissions_inventory/national_state/process.py

```diff
@@ -101,6 +101,9 @@ def _add_sv_and_mcf_column_to_final_df(self):
         }
 
         sv_list = pd.unique(self._final_df['SV'])
+        #sv_list = sv_list[np.logical_not(np.isnan(sv_list))]
```

Comment:
Please delete commented lines (here and elsewhere)

Reply:
I have removed all commented lines